apps/drupal-default/particle_theme/templates/block/block--utc-highlights-block.html.twig

```diff
@@ -0,0 +1,164 @@
+{#
+/**
+ * @file Card Grid!
+ * Implements a hero block in dark blue
+ *
+ *
+ * Available variables:
+ */
+ *
+ * This template pulls in the variables to add to the cards.
+ * References the block--utc-card-base.html.twig to create the individual cards in this grid.
+#}
+
+{% block content %}
+  {# Sets variable to trigger content render array. #}
+  {% set rendered_content = content|render %}
+
+  {% set highlight_image_select = content["#block_content"].field_highlight_info_bg.0.target_id %}
+  {% set utc_highlight = {
+    highlight_bg: drupal_field('field_image_url', 'taxonomy_term', highlight_image_select)["#object"].field_image_url.value,
+    style: content["#block_content"].field_highlight_info_style.0.value,
+    bg_color: content["#block_content"].field_highlight_background_color.0.value,
+    title: content.field_highlight_info_title|field_value,
+  } %}
+  {% set info_1 = content.field_highlight_info_1|field_value %}
+  {% set info_2 = content.field_highlight_info_2|field_value %}
+
+  {# Sets colors for hero content depending on selection #}
+  {% if utc_highlight.bg_color == 'dark' %}
+    {% set info_color = 'text-utc-new-gold-500' %}
+    {% set info_color_2 = 'text-utc-new-blue-500' %}
+    {% set icon_color = 'text-white' %}
+    {% set icon_color_2 = 'text-utc-new-blue-500' %}
+    {% set description_color = 'text-white' %}
+    {% set description_color_2 ='text-utc-new-blue-500' %}
+    {% set bg_color = 'background-color:#112e51' %}
+    {% set bg1_opacity = '' %}
+    {% set bg2_opacity = 'opacity-90' %}
+    {% set bg_color_2 = 'rgba(226, 232, 240, 0.9)' %}
+  {% else %}
+    {% set info_color = 'text-utc-new-blue-500' %}
+    {% set info_color_2 = 'text-utc-new-gold-500' %}
+    {% set icon_color = 'text-utc-new-blue-500' %}
+    {% set icon_color_2 = 'text-white' %}
+    {% set description_color = 'text-utc-new-blue-500' %}
+    {% set description_color_2 ='text-white' %}
+    {% set bg_color = 'background-color:#104dd4' %}
+    {% set bg1_opacity = 'opacity-20' %}
+    {% set bg2_opacity = 'opacity-20' %}
+    {% set bg_color_2 = 'rgba(17, 46, 81, 0.9)' %}
+  {% endif %}
+
+  {% if utc_highlight.style == 'Side Box' %}
+    <div
+      class="grid grid-cols-1 lg:grid-cols-utchighlight lg:grid-rows-utchighlight lg:gap-y-4 lg:gap-x-10 grid-flow-row">
+      <div
+        class="col-start-1 row-start-1 m-10 mt-20 lg:m-0 lg:col-start-2 lg:col-end-3 lg:row-start-2 lg:row-end-4 lg:mb-10 z-10">
+        {% if utc_highlight.title %}
+          <h2 class="pl-6 {{ description_color }}">{{ utc_highlight.title }}</h2>
+          <hr class="border-t-4 border-utc-new-gold-500 mt-3 mb-6 ml-8 w-20"/>
+        {% endif %}
+        <ul class="flex flex-row flex-wrap list-none pl-0 mb-0">
+          {% for content in info_1.0['#icons'] if info_1.0['#icons'] is not empty %}
+            <li class="p-10 text-lg text-center xl:w-1/2 w-full {{ description_color }}">
+              {% if info_1.0['#icons'][loop.index0]['#name'] %}
```

Comment:
It probably be adding to this, as an if else to check on whether there's none then higher font size.

Reply:
Added!